lib/utils.js

```diff
@@ -0,0 +1,82 @@
+/*
+ * Kuzzle, a backend software, self-hostable and ready to use
+ * to power modern apps
+ *
+ * Copyright 2015-2020 Kuzzle
+ * mailto: support AT kuzzle.io
+ * website: http://kuzzle.io
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+'use strict';
+
+/**
+ * 
+ * @param {any} object Object to test
+ * @returns {boolean} true if it's a plain object, false otherwise
+ */
+function isPlainObject (object) {
```

Comment:
Not related to this PR but maybe we could expose this helpers from Kuzzle core in a `SafeObject` static class (like the `Inflector` module)

Reply:
I agree, there is the feature request kuzzleio/kuzzle#2072